database_version_8.sql

```diff
@@ -0,0 +1,3 @@
+alter table CURRENCYFORMATS_V1 add column CURRENCY_TYPE TEXT;
+update CURRENCYFORMATS_V1 set CURRENCY_TYPE = 'Traditional';
```

Comment:
https://en.m.wikipedia.org/wiki/Fiat_money
I suggest to use more simple string for non crypto currencied : 'fiat'